qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -49,16 +61,22 @@
     variants:
         - gagent_install:
             gagent_check_type = install
+            Windows:
+                check_vioser = yes
```

Comment:
You don't need this, and below ones till Line 510.

Reply:
oh, you mean I set check_vioser = yes at line30 so that it's not necessary to set them at each case beginning right?

Reply:
Yes, right.